docs/attribution.mdx

```diff
@@ -19,121 +19,134 @@ The Overture Maps Foundation does not require text attribution or an OMF logo on
 #### Austria
-- © [Austrian Address Register](https://geometadatensuche.inspire.gv.at/metadatensuche/srv/ger/catalog.search#/metadata/bd28c2b0-4ea5-41c2-8847-b7e8711a6512). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Distributed by OpenAddresses. Accessed: 2025-01-20.
+- © [Austrian Address Register](https://geometadatensuche.inspire.gv.at/metadatensuche/srv/ger/catalog.search#/metadata/bd28c2b0-4ea5-41c2-8847-b7e8711a6512). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Distributed by OpenAddresses. Accessed: 2025-02-16.
 #### Belgium
-- Wallonia data from [ICAR](https://opendata.bosa.be/index.fr.html). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Accessed: 2025-01-20.
-- Flanders data from [AR (Adressenregister)](https://opendata.bosa.be/index.fr.html). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Accessed: 2025-01-20.
-- Brussels data from [UrbiS](https://opendata.bosa.be/index.fr.html). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Accessed: 2025-01-20.
+- Wallonia data from [ICAR](https://opendata.bosa.be/index.fr.html). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Accessed: 2025-02-16.
+- Flanders data from [AR (Adressenregister)](https://opendata.bosa.be/index.fr.html). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Accessed: 2025-02-16.
+- Brussels data from [UrbiS](https://opendata.bosa.be/index.fr.html). Available under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/). Accessed: 2025-02-16.
 #### Brazil
 - [IBGE (Instituto Brasileiro de Geografia e Estatística)](https://www.ibge.gov.br/). Available under [CC0](https://creativecommons.org/public-domain/cc0/). Distributed by AddressForAll. Accessed: 2024-10-13
 #### Canada
-- [Statistics Canada, National Address Register](https://www150.statcan.gc.ca/n1/pub/46-26-0002/462600022022001-eng.htm), 2024. Reproduced and distributed on an "as is" basis with the permission of Statistics Canada. Available under [OGL - Statistics Canada Open Licence](https://www.statcan.gc.ca/en/reference/licence). Distributed by OpenAddresses. Accessed: 2025-01-20.
+- [Statistics Canada, National Address Register](https://www150.statcan.gc.ca/n1/pub/46-26-0002/462600022022001-eng.htm), 2024. Reproduced and distributed on an "as is" basis with the permission of Statistics Canada. Available under [OGL - Statistics Canada Open Licence](https://www.statcan.gc.ca/en/reference/licence). Distributed by OpenAddresses. Accessed: 2025-02-16.
+- Yukon data from [Contains information licensed under the Open Government Licence – Yukon](https://map-data.service.yukon.ca/GeoYukon/People_and_Structures/Civic_Address_Point/). Available under [Open Government Licence - Yukon](https://open.yukon.ca/open-government-licence-yukon). Distributed by OpenAddresses. Accessed: 2025-02-16.
```

Comment:
Small nit: maybe this should say "Yukon data from [GeoYukon]" or similar?

Reply:
Ah yeah good call. Looks like the rest server uses "Government of Yukon" as the copyright text so we'll go with that.